src/main/java/core/basesyntax/service/transaction/TransactionHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.transaction;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitInventoryManipulator;
+
+public interface TransactionHandler extends FruitInventoryManipulator {
```

Comment:
I can't see reasons why TransactionHandler should extend FruitInventoryManipulator.
They have different purposes.
Implementations of TransactionHandler should use FruitInventoryManipulator as instrument